packages-next/keystone/src/lib/itemAPI.ts

```diff
@@ -34,27 +34,35 @@ export function getArgsFactory(list: BaseKeystoneList, schema: GraphQLSchema) {
   };
 }
 
+function defaultQueryParam(query?: string | false, resolveFields?: string | false) {
+  if (resolveFields !== undefined) return resolveFields;
+  if (query !== undefined) return query;
```

Comment:
We should prefer query or throw an error if both are passed

Reply:
Good point.

Real q: what would be the right way of throwing an error in this part of the codebase?

Reply:
`throw new Error('')`?

Reply:
I have updated it but lmk if we should be throwing errors differently

Reply:
I think you forgot to push?